### PR TITLE
First draft of the contribution guidelines

### PR DESCRIPTION
**Pull request by yaustar**
First pass on the contribution guidelines

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Other files such as assets, zips can be placed in [`public downloads`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/downloads). As this is used very rarely, there is no folder structure to maintain.
+
+### External projects
+
+Usually tutorials require a PlayCanvas project. Please transfer ownership of any related projects to user `playcanvas`. Steps to transfer projects can be [found here in the documentation](https://developer.playcanvas.com/en/user-manual/profile/projects/#transfer-project-ownership).
+
+### Code examples
+
+Please refer to the [Engine contribution guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) for code styling. Use Javascript ES5 as the code is intended to be ran directly in the browser.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:67]:
```suggestion
Please refer to the [Engine contribution guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) for code styling. Use JavaScript ES5 as the code is intended to be run directly in the browser.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+GIFs can be used but please keep the file size as low as possible. No larger than 2MB.
+
+### Other files
+
+Other files such as assets, zips can be placed in [`public downloads`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/downloads). As this is used very rarely, there is no folder structure to maintain.
+
+### External projects
+
+Usually tutorials require a PlayCanvas project. Please transfer ownership of any related projects to user `playcanvas`. Steps to transfer projects can be [found here in the documentation](https://developer.playcanvas.com/en/user-manual/profile/projects/#transfer-project-ownership).
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:63]:
```suggestion
Usually, tutorials require a PlayCanvas project. Please transfer ownership of any related projects to user `playcanvas`. Steps to transfer projects can be [found here in the documentation](https://developer.playcanvas.com/en/user-manual/profile/projects/#transfer-project-ownership).
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Images should be placed in the [`public/images`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/images) folder. The folder structure should mirror the same structure as the folder for the markdown files mentioned in [Pages section](#Pages).
+
+Please use PNG for anything Editor UI based and JPEG for anything else such as a game screenshot.
+
+GIFs can be used but please keep the file size as low as possible. No larger than 2MB.
+
+### Other files
+
+Other files such as assets, zips can be placed in [`public downloads`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/downloads). As this is used very rarely, there is no folder structure to maintain.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:59]:
```suggestion
Other files such as assets and zips can be placed in [`public downloads`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/downloads). As this is used very rarely, there is no folder structure to maintain.
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+- `<iframe>` for PlayCanvas project builds
+- `<img>` to adjust sizes of images
+- alert style boxes
+
+Refer to existing pages to see examples of prior use.
+
+### Images
+
+Images should be placed in the [`public/images`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/images) folder. The folder structure should mirror the same structure as the folder for the markdown files mentioned in [Pages section](#Pages).
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:51]:
```suggestion
Images should be placed in the [`public/images`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/public/images) folder. The folder structure should mirror the same structure as the folder for the markdown files mentioned in the [Pages section](#Pages).
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Avoid using acronyms unless they are already defined on the page.
+
+### Use International/American English spelling
+
+For example, use "Initialize" instead of "Initialise", and "color" instead of "colour".
+
+### Pages
+
+Each page on [developer.playcanvas.com](developer.playcanvas.com) is generated from a markdown file in [`content`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/content). English (en) will always be the first version and PlayCanvas has external services to translate to other languages.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:35]:
```suggestion
Each page on [developer.playcanvas.com](https://developer.playcanvas.com) is generated from a Markdown file in [`content`](https://github.com/playcanvas/developer.playcanvas.com/tree/master/content). English (en) will always be the first version and PlayCanvas has external services to translate to other languages.
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+6. Complete the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).
+7. Happy Days! 🎉
+
+### Tips
+
+Feel free to contribute documentation fixes and new tutorials as a pull request.
+
+If you are looking for ideas what to work on, head to [Issues](https://github.com/playcanvas/developers/issues) and checkout out open tickets or start a conversation. It is best to start conversation if you are going to create a larger tutorial or refactor a section of the user manual. [Forum](http://forum.playcanvas.com/) is good place to have a chat with community as well.
+
+Try to keep PR focused on a single subject or tutorial, small PR's are easier to review and will get merged faster.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:19]:
```suggestion
Try to keep PR focused on a single subject or tutorial. Small PRs are easier to review and will get merged faster.
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+4. Keep PR simple and focused - one PR per feature with a screenshot of the changes on the site. [Example PR][https://github.com/playcanvas/developer.playcanvas.com/pull/250].
+5. Make a Pull Request.
+6. Complete the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).
+7. Happy Days! 🎉
+
+### Tips
+
+Feel free to contribute documentation fixes and new tutorials as a pull request.
+
+If you are looking for ideas what to work on, head to [Issues](https://github.com/playcanvas/developers/issues) and checkout out open tickets or start a conversation. It is best to start conversation if you are going to create a larger tutorial or refactor a section of the user manual. [Forum](http://forum.playcanvas.com/) is good place to have a chat with community as well.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:17]:
```suggestion
If you are looking for ideas what to work on, head to [Issues](https://github.com/playcanvas/developers/issues) and check out open tickets or start a conversation. It is best to start conversation if you are going to create a larger tutorial or refactor a section of the user manual. The [Forum](http://forum.playcanvas.com/) is good place to have a chat with the community as well.
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
+# CONTRIBUTING
+
+# HOW TO CONTRIBUTE
+
+1. Looking for ideas? Check out ["good first PR"](https://github.com/playcanvas/developer.playcanvas.com/labels/good%20first%20PR) label.
+2. Or start a conversation in [Issues](https://github.com/playcanvas/developer/issues) to get help and advice from community on PR ideas.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:6]:
```suggestion
2. Or start a conversation in [Issues](https://github.com/playcanvas/developer/issues) to get help and advice from the community on PR ideas.
```

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+3. Read the writing standards below.
+4. Keep PR simple and focused - one PR per feature with a screenshot of the changes on the site. [Example PR][https://github.com/playcanvas/developer.playcanvas.com/pull/250].
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:8]:
```suggestion
4. Keep PRs simple and focused - one PR per feature with a screenshot of the changes on the site. [Example PR][https://github.com/playcanvas/developer.playcanvas.com/pull/250].
```

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+If you are looking for ideas what to work on, head to [Issues](https://github.com/playcanvas/developers/issues) and checkout out open tickets or start a conversation. It is best to start conversation if you are going to create a larger tutorial or refactor a section of the user manual. [Forum](http://forum.playcanvas.com/) is good place to have a chat with community as well.
+
+Try to keep PR focused on a single subject or tutorial, small PR's are easier to review and will get merged faster.
+
+# WRITING STANDARDS
+
+### General
+
+Keep in mind of your target audience and assume that they are completely new to subject or PlayCanvas. Refer them to other pages or tutorials if you need prior knowledge to better understand the current content. For example, an FPS tutorial can refer to Basic Keyboard and Mouse input as tutorials to complete before continuing.
```
Comment (willeastcott) [inline .github/CONTRIBUTING.md:25]:
```suggestion
Keep your target audience in mind and assume that they are completely new to the subject or PlayCanvas. Refer them to other pages or tutorials if you need prior knowledge to better understand the current content. For example, an FPS tutorial can refer to Basic Keyboard and Mouse Input as tutorials to complete before continuing.
```

**Review by willeastcott — APPROVED** [review]
Approving with a few minor tweaks. Nice!

**Comment by yaustar** [comment]
Thanks Will. Some of this was copy and pasted from the engine repro so it also may have some of these typos in

**Pull request merged**